thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -203,17 +265,21 @@ export default Component.extend({
      * Updates the currently selected urns based on user selection on the table
      * @param {Object} e
      */
-    displayDataChanged (e) {
-      if (_.isEmpty(e.selectedItems)) { return; }
+    displayDataChanged(e) {
+      if (_.isEmpty(e.selectedItems)) {
+        return;
+      }
 
       const { selectedUrns, onSelection } = getProperties(this, 'selectedUrns', 'onSelection');
 
-      if (!onSelection) { return; }
+      if (!onSelection) {
+        return;
+      }
```

Comment:
You can optimize the code.

if(_.isEmpty(e.selectedItems) || !onSelection) return